.github/workflows/test.yml

```diff
@@ -4,11 +4,13 @@ on:
   push:
     branches:
       - "master"
```

Comment:
Are you going to drop master separately?

Reply:
just preparing the switch, not doing directly after